### ISSUE TITLE
NVDA too verbose in Gmail settings

### ISSUE THREAD

**Issue by nimerj** [nvaccess/nvda#9645]
### Steps to reproduce:
1. Open Chrome and navigate to gmail.com
2. Navigate to gmail settings by pressing tab to settings and pressing enter, down arrow to settings and pressing enter again.
3. tab throughout the settings dialog through the various options. Tab ten or more times to really observe the issue, and listen to what NVDA verbalizes
### Actual behavior:
NVDA is entirely too verbose, reading settings options at least twice, announcing tables, announcing the link following the settings item if there is one, etc. As an example, Always display external images - Learn more about display of external images by default Ask before displaying external images - This option also disables dynamic email.  row 6  column 2
table  row 6
Always display external images  row -1
Always display external images  radio button  checked  row -1  1 of 2
Always display external images - Learn more about display of external images by default  row -1
Learn more about display of external images by default  link  row -1

### Expected behavior:
NVDA announces options once, and does not announce the link following the setting. This is the behavior with JAWS and Chrome, as well as with Firefox and NVDA/JAWS.
#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
Alpha 17384
#### Windows version:
1809
#### Name and version of other software in use when reproducing the issue:
chrome Canary Version 76.0.3808.0 
#### Other information about your system:
Lenovo Thinkpad laptop
#### Does the issue still occur after restarting your PC?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no

**Comment by Adriani90** [comment]
I think this is an implementation error with Google Mail itself, it seems NVDA reports all available radio button options when tabbing to a setting and then announces the selected option. @aleventhal I think this might be due to groupings description on settings which contains detailed information leading to double speaking. But not sure this is related to Chrome or NVDA.
Should I create an issue on Chromium bug tracker for this?

**Comment by Adriani90** [comment]
I created https://issues.chromium.org/issues/349393381
